test-cases/longevity/longevity-gce-custom-d1-workload2-hybrid-raid.yaml

```diff
@@ -46,313 +47,284 @@ prepare_write_cmd:
   # '800', '1600', '3200' and '4400'.
   # TODO: If DB cluster gets extended or summary load gets reduced reconsider the large partition sizes.
   - >-
-    latte run --tag latte-prepare-01 --duration 37550050 --request-timeout 60 --retry-interval '500ms,5s'
-    --sampling 5s --threads 30 --connections 3 --concurrency 180 --rate 9500 -P offset=0
-    --function custom -P codes="\"T14F10\"" -P row_count=75100100 -P batch_size=2
+    latte run --tag latte-prepare-01 --duration 75100100 --request-timeout 60 --retry-interval '1s,5s'
```

Comment:
why are we doubling the data here ? isn't this change is just renaming things ?

Reply:
> Increase nemesis interval from 5 to 10 minutes to mitigate not finished operations (compactions) left after impactful nemesis such as "disrupt_destroy_data_then_repair".

now I saw, should this be a SCT issue ? maybe this nemesis need to wait for something it's not waiting now ?

Reply:
seems like lots of reshuffling in the prepare commands, can you clarify why ? what actually changed ?

Reply:
Not doubling, combining 2 separate command into one.
Before it was written in 2 parts by 2 loaders.
In this change all the data get written with just 1 latte command.